Source/MultipartFormData.swift

```diff
@@ -256,16 +256,17 @@ public class MultipartFormData {
 
         var isReachable = false
 
-        do {
-            try fileURL.checkResourceIsReachable()
+        if #available(OSX 10.10, *) {
```

Comment:
The main reason we're dropping iOS 7 support in Alamofire 2.0 is that we can't test it. We don't want to, nor do we have the bandwidth to re-write our entire test suite to support iOS 7 since it doesn't support frameworks. Additionally,  Travis-CI does not support iOS 7 leaving us in a position where it just doesn't make sense to support it. This is just a decision that we've made for the greater good overall.

With that said, there's no reason you can't run a fork for yourself to make the changes. It's just not something we're officially going to support moving forwards.

Cheers.